cadc-uws-server/src/main/java/ca/nrc/cadc/uws/web/restlet/RestletPrincipalExtractor.java

```diff
@@ -127,13 +165,25 @@ private void init()
                 finally { }
             }
         }
+
+        // add HttpPrincipal
+        final String httpUser = getAuthenticatedUsername();
+        if (StringUtil.hasText(httpUser)) // user from HTTP AUTH
+            principals.add(new HttpPrincipal(httpUser));
+        else if (token != null) // user from token
+            principals.add(token.getUser());
```

Comment:
Why is only the one identity from the token added?  Should these be treated the same as cookies?